module/solutions/helper.js

```diff
@@ -1993,44 +2012,77 @@ module.exports = class SolutionsHelper {
 					})
 				}
 
-				if (solutionData[0].scope.entityType !== programData[0].scope.entityType) {
-					let checkEntityInParent = await entitiesService.entityDocuments(
-						{
-							_id: programData[0].scope.entities,
-							[`groups.${solutionData[0].scope.entityType}`]: entities,
-						},
-						['_id']
-					)
-					if (!checkEntityInParent.success) {
-						throw {
-							message: CONSTANTS.apiResponses.ENTITY_NOT_EXISTS_IN_PARENT,
-						}
-					}
-				}
+				// This logic we need to re-look --------------------------------------------
```

Comment:
@MallanagoudaB hence we have dynamic scoping structure this logic has been commented for now we have to check this for enhancement if required